pretest.js

```diff
@@ -1,23 +1,25 @@
 'use strict';
-// TODO: used for testing support for parallel testing on ci.strongloop.com which
-// provides MYSQL_* env vars instead of TEST_MYSQL_* env vars.
-process.env.TEST_MYSQL_USER = process.env.TEST_MYSQL_USER || process.env.MYSQL_USER;
-process.env.TEST_MYSQL_PASSWORD = process.env.TEST_MYSQL_PASSWORD || process.env.MYSQL_PASSWORD;
-process.env.TEST_MYSQL_HOST = process.env.TEST_MYSQL_HOST || process.env.MYSQL_HOST;
-process.env.TEST_MYSQL_PORT = process.env.TEST_MYSQL_PORT || process.env.MYSQL_PORT;
 
-if (!process.env.TEST_MYSQL_USER) {
-  console.log('not seeding DB with example db');
-  return;
+if (!process.env.TEST_MYSQL_USER && !process.env.CI) {
+  return console.log('Not seeding DB with test db');
 }
 
 var fs = require('fs');
 var cp = require('child_process');
 
 var sql = fs.createReadStream(require.resolve('./test/schema.sql'));
 var stdio = ['pipe', process.stdout, process.stderr];
-var args = ['--user=' + process.env.TEST_MYSQL_USER];
 
+process.env.TEST_MYSQL_HOST =
```

Comment:
Putting these above the first `require` would give a smaller diff and not require the `var` lines to be split up.